mixins/property-required/property-required-mixin.js

```diff
@@ -0,0 +1,107 @@
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+
+const TIMEOUT_DURATION = 3000;
+export const ERROR_CODE = '(@brightspace-ui/core:PropertyRequiredMixin)';
```

Comment:
This is the first time we've done this, but what do folks think about including an error code in the exceptions that we throw? This would allow us to search JavaScript errors in the logs for `@brightspace-ui/core:` or for errors thrown from this mixin specifically. Because the error messages contain tag and property names, it would be difficult to search for them otherwise.